pkg/configure/root.go

```diff
@@ -39,7 +39,7 @@ func NewRootCommand(parent common.Registerer, configFilePath string, cf APIClien
 }
 
 // Exec implements the command interface.
-func (c *RootCommand) Exec(in io.Reader, out io.Writer) error {
+func (c *RootCommand) Exec(in io.Reader, out io.Writer) (err error) {
```

Comment:
The `var err error` on line 68 now shadows this error variable, so that case will not trigger progress.Fail().

I generally don't like named return variables unless they support documentation readability from afar, and `error` is so ubiquitous in Go that that isn't really necessary. If it's just here for the side effect of getting the variable declared before the defer(), the intent here may be clearer as a `var err error` at the top of the function.